maestro/src/test/java/org/intocps/maestro/JacobianStepBuilderTest.java

```diff
@@ -0,0 +1,144 @@
+package org.intocps.maestro;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.provider.Arguments;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.stream.Stream;
+
+import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
+
+public class JacobianStepBuilderTest extends FullSpecTest {
```

Comment:
Maybe some of the functionality here is general to many different tests? Do nothing, but think of it